src/chttpd/src/chttpd.erl

```diff
@@ -1009,6 +1011,10 @@ error_info({request_entity_too_large, {bulk_get, Max}}) when is_integer(Max) ->
     {413, <<"max_bulk_get_count_exceeded">>, integer_to_binary(Max)};
 error_info({request_entity_too_large, DocID}) ->
     {413, <<"document_too_large">>, DocID};
+error_info({request_transcation_too_large, Code}) ->
+    CodeBin = integer_to_binary(Code),
+    {413, <<"request_transcation_too_large">>,
+        <<"The request transcation is too large. (", CodeBin/binary, ")" >>};
```

Comment:
Users might confuse `2101` with the transaction size limit. Since we know the limit as 10MB we can just include in the message perhaps something like `{413, <<"transaction_too_large">>, <<"The request transaction is larger than 10MB">>}`